wandb/sdk/wandb_run.py

```diff
@@ -489,7 +488,6 @@ def _init(
         self._exit_code = None
```

Comment:
love to see this change! so much cleaner thanks!!

Reply:
:)